setup/setup.iss

```diff
@@ -351,7 +346,7 @@ end;
 
 procedure ReplaceRedFactionLauncher;
 begin
-    if IsTaskSelected('replacerflauncher') then
+    if WizardIsTaskSelected('replacerflauncher') then

```

Comment:
this is Inno Setup 6.x function, but I guess we can migrate from 5.x now that I stopped supporting windows xp...

Reply:
I was pondering about this. I initially used 5.6.1 to test the installer on XP and it did run, but Dash itself no longer opens so I bumped up to 6.2.1. Another thing that changed in 6.x was the `{pf}` constant, which is now `{commonpf}`, though the original naming does still work.